packages/flutter/lib/src/rendering/slivers.dart

```diff
@@ -0,0 +1,1850 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
Ah ha! No! I'm ahead of you on this one. I wrote the code and published it on github in 2016! :-P